src/context/GameContext.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+import { useContext } from 'react';
+import { createContext } from 'react';
+
+const GameContext = createContext();
+
+
+
+
+export function GameContextProvider({ children }) {
+  const [square, setSquares] = useState(Array(9).fill(''));
```

Comment:
Ok, nothing wrong here, I actually wanted to say, I really like the way you set the default state using old array instantiation combined with an array method. Good work there!